extensions/extensions-docker/pom.xml

```diff
@@ -18,37 +18,43 @@
 
     <profiles>
         <profile>
-            <id>x86</id>
-            <properties>
-                <target.architecture>amd64</target.architecture>
-                <docker.platforms>linux/amd64</docker.platforms>
-            </properties>
-        </profile>
-        <profile>
-            <id>arm64</id>
-            <properties>
-                <target.architecture>arm64</target.architecture>
-                <docker.platforms>linux/arm64</docker.platforms>
-            </properties>
+            <id>ci</id>
             <build>
-                <pluginManagement>
-                    <plugins>
-                        <plugin>
-                            <artifactId>maven-deploy-plugin</artifactId>
-                            <configuration>
-                                <skip>true</skip>
-                            </configuration>
-                        </plugin>
-                    </plugins>
-                </pluginManagement>
+                <plugins>
+                    <plugin>
+                        <groupId>io.fabric8</groupId>
+                        <artifactId>docker-maven-plugin</artifactId>
+                        <executions>
+                            <execution>
+                                <id>default-push</id>
+                                <configuration>
+                                    <images>
+                                        <image>
+                                            <name>${docker.baseline.repo.id}/${dockerImageName}:${dockerImageVersion}</name>
+                                            <build>
+                                                <buildx>
+                                                    <platforms>
+                                                        <platform>${docker.platforms}</platform>
+                                                    </platforms>
+                                                </buildx>
+                                                <args>
+                                                    <DELTA_HIVE_CONNECTOR_VERSION>${version.delta.hive.connector}</DELTA_HIVE_CONNECTOR_VERSION>
+                                                    <DOCKER_BASELINE_REPO_ID>${docker.baseline.repo.id}/</DOCKER_BASELINE_REPO_ID>
+                                                    <VERSION_AISSEMBLE>${version.aissemble}</VERSION_AISSEMBLE>
+                                                    <SPARK_VERSION>${version.spark}</SPARK_VERSION>
+                                                </args>
+                                                <contextDir>${project.basedir}</contextDir>
+                                                <dockerFile>./src/main/resources/docker/Dockerfile</dockerFile>
```

Comment:
Is this configuration needed? Looks the same as the config specified in the `build-parent/pom.xml`.

Reply:
yes, unfortunately.

Reply:
Oh this is `<id>default-push</id>` instead of `<id>default-build</id>, so makes since to have to duplicate them.